### ISSUE TITLE
Example standard-compliant files

### ISSUE THREAD

**Issue by amlalejini** [alife-data-standards/alife-data-standards#10]
Should we have example files (.csv, .json, etc) for each data type that can be used for tool validation, as a guide for data converter developers, etc?

If so, we should talk about the specifics of each. E.g., how do we want to format lists of information in a csv?

**Comment by cliff-bohm** [comment]
I agree we need examples. I think we can give examples of the phylogeny standard at the moment.
We should do something smart about csv lists perhaps "[x,y,z,[a,b,c]]" (quote + bracket opens and then brackets are counted).

**Comment by mmore500** [comment]
Another tool that might be useful to implementors would be validators (i.e., code that takes a file and says whether or not it is a valid instance of a standard). If elements of the standard are deprecated or changed over time, these validators could be updated to issue warnings or errors with specific information about the changes.

**Comment by mmore500** [comment]
For the benefit of future browsers, there are some example files available here: <https://github.com/alife-data-standards/alife-std-dev-python/tree/f21a63d70077345441b9b52c3f470f5dca1127c1/example_data/>.
